packages/gatsby/content/advanced/migration.md

```diff
@@ -37,9 +37,27 @@ Run `npx @yarnpkg/doctor .` (or `yarn dlx @yarnpkg/doctor .`) in your project to
 
 Note that the doctor is intended to report any potential issue - it's then up to you to decide whether they are a false positive or not (for example it won't traverse Git repositories). For this reason we don't recommend using it as a CI tool.
 
-### Use `yarn dlx` instead of `yarn global`
-
-`yarn dlx` is designed to execute one off scripts that may have been installed as global packages with `yarn 1.x`. Managing system-wide packages is outside of the scope of `yarn`. To reflect this, `yarn global` has been removed. [Read more on GitHub](https://github.com/yarnpkg/berry/issues/821).
+### Changes in CLI commands
+
+| v1              | v2 substitute                | Notes                                                               |
+| --------------- | ---------------------------- | ------------------------------------------------------------------- |
+| `yarn audit`    | -                            | [Read more on GitHub](https://github.com/yarnpkg/berry/issues/1187) |
+| `yarn check`    | `yarn install --check-files` | [Read more on GitHub](https://github.com/yarnpkg/rfcs/pull/106)     |
+| `yarn create`   | -                            | Not implemented                                                     |
```

Comment:
But `yarn create` is not documented. And I don't think the migration guide should refer to undocumented behavior. Should I add a document for `yarn create` or change to `yarn dlx create-<package-name>` here?

Reply:
We could mention that it's implemented as an alias to `yarn dlx`. Even if it's hidden, once you document it, it's not undocumented behavior anymore. 😄
I don't think that mentioning it would cause any harm, since it can be considered a useful shorthand and it will continue to be supported in the future. 🙂

Reply:
> it can be considered a useful shorthand and it will continue to be supported in the future.

Maybe we can just add it to the documentation of CLI commands then?

Reply:
I'd prefer not. It's aliased to make the migration easier (so no need to mention it in the migration guide), but going forward I prefer people to use `dlx` (hence why it's hidden from the CLI doc).

Reply:
updated. See if you like the new version more.